src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -45,95 +39,103 @@
       - sudo
       - systemd
       - tcpdump
-      - tig
-      - tmate
-      - tmux
-      - vim
       - wget
       - which
       - wireshark-cli
+      disable_gpg_check: yes
```

Comment:
Why? This definitely should not be needed.

Reply:
This is needed for internal repos.